vcluster/_partials/deploy/deploy.mdx

```diff
@@ -173,14 +175,147 @@ Commit and push these files to your configured ArgoCD repository.
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: HelmRepository
+metadata:
+  name: vcluster
+  namespace: flux-system
+spec:
+  interval: 1h
+  url: https://charts.loft.sh
+```
+
+</Step>
+
+<Step>
+The vCluster will be deployed into the namespace `team-x` of the host cluster. If this namespace doesn’t already exist, create it with:
+```bash title="Create the namespace"
+kubectl create namespace team-x
+```
+
+</Step>
+
+<Step>
+
+Create a vCluster `HelmRelease` file in your Git repository. This `HelmRelease` tells Flux how to deploy a vCluster in your Kubernetes cluster using the configured Helm charts in Step 2:
+
+```yaml title="clusters/production/vcluster-demo.yaml"
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: vcluster-demo
+  namespace: team-x
+spec:
+  interval: 10m
+  chart:
+    spec:
+      chart: vcluster
+      version: "0.24.x"
+      sourceRef:
+        kind: HelmRepository
+        name: vcluster
+        namespace: flux-system
+  values:
+    # Configure TLS SAN for the certificate
+    controlPlane:
+      proxy:
+        extraSANs:
+          - "vcluster-demo.team-x.svc.cluster.local"
+      coredns:
+        enabled: true
+
+    exportKubeConfig:
+      # Set a meaningful context name
+      context: default
+      # Use a server URL that matches the TLS SAN
+      server: https://vcluster-demo.team-x.svc.cluster.local:443
+      # Skip TLS verification when Flux connects to the vCluster
+      insecure: true
+      # Specify the secret where the KubeConfig is stored
+      secret:
+        name: vcluster-flux-kubeconfig
+    sync:
+      toHost:
+        ingresses:
+          enabled: true
+      
+```
+The content in `vcluster.yaml` should be placed under the `values`.
+
+Unlike ArgoCD, Flux requires the KubeConfig secret to be able to access the vCluter's API server and deploy to vCluster. 
+The `exportKubeConfig` configuration under `values`:
+- Exports the virtual cluster KubeConfig as a Secret in the host namespace
+- Makes the Secret available for Flux to use with the `spec.kubeConfig` field
+- Uses a server URL that is accessible from the Flux controllers (replace `vcluster-name` and `team-x` with your actual values)
+- Sets `insecure: true` to automatically skip TLS certificate verification
+- Adds a TLS SAN (Subject Alternative Name) that matches the server URL, which helps prevent certificate verification errors
+
+:::info KubeConfig Secret key
+The vCluster `exportKubeConfig` configuration creates a Secret with the KubeConfig data stored under the key `config`. When referring to this Secret in Flux resources, you must specify this key in the `secretRef.key` field, as shown in the examples below.
+
+```yaml
+# In Flux HelmRelease
+spec:
+  kubeConfig:
+    secretRef:
+      name: vcluster-flux-kubeconfig
+      key: config  # Must match the key used in the vCluster-generated Secret
+```
+:::
+
+:::warning Certificate verification considerations
```

Comment:
This renders incorrectly 

<img width="655" height="173" alt="Image" src="https://github.com/user-attachments/assets/fdfd983a-0a04-4076-bcaa-ae439393ffd9" />